### ISSUE TITLE
Local labels are also valid for EQU declaration

### ISSUE THREAD

**Issue by Farzy** [dude719/JetBrains-NASM-Language#33]
Hi,

Using a sample "Hello, world" ASM code I found on Internet, I discovered that using a [local label](https://www.nasm.us/xdoc/2.15.04/html/nasmdoc3.html#section-3.9), "`.len`", on a "`EQU`" statement seems to be valid. The following code compiles with NASM 2.15.05 and works:

<img width="1591" alt="Capture d’écran 2020-08-28 à 18 25 08" src="https://user-images.githubusercontent.com/41388/91590636-f29ea380-e95b-11ea-9822-7fc2245e4553.png">

Your plugin flags it as invalid. Is it something you can change?

Regards

**Comment by ajkhoury** [comment]
Yeah this is a known problem with the lexer, for a quick workaround just add a colon (:) at the end of .len like so:

`.len: equ $-message`

**Comment by Farzy** [comment]
You mean a colon? I already tried before opening this ticket, it only moved the error on `equ` :)

<img width="825" alt="Capture d’écran 2020-08-28 à 22 18 50" src="https://user-images.githubusercontent.com/41388/91611979-bd567d80-e97c-11ea-9687-12d0e466eae0.png">

**Comment by ajkhoury** [comment]
Yeah I meant a colon haha! I'll try to find some spare time to tinker with the lex later today.
